torchgeo/datamodules/chesapeake.py

```diff
@@ -169,24 +169,34 @@ def preprocess(self, sample: Dict[str, Any]) -> Dict[str, Any]:
         Returns:
             preprocessed sample
         """
-        sample["image"] = sample["image"] / 255.0
```

Comment:
I changed all `sample["image"] = sample["image"] / 255.0` to `sample["image"] /= 255.0` as the inline operation shouldn't make any copies.